raiden/ui/app.py

```diff
@@ -270,7 +270,9 @@ def run_app(
             click.secho(
                 f"'{pathfinding_service_info['message']}'. "
                 f"You have chosen pathfinding operator '{pathfinding_service_info['operator']}' "
```

Comment:
This information should ideally also be logged. E.g. Could just be a simple `log.info('Using PFS', pfs_info=pathfinding_service_info)`

Reply:
thanks done